.ci/.e2e-platforms.yaml

```diff
@@ -45,6 +45,11 @@ PLATFORMS:
     image: "ami-0d90bed76900e679a"
     instance_type: "t3.xlarge"
     username: admin
+  macos12:
+    description: "Mac OS 12 platform"
+    image: "ami-094dfaad20f351cb7"
```

Comment:
@jlind23 , where did you find this `ami` image? If you can provide those details please,